# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-backgrounds/animations/background-color-animation-with-table3.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/css-backgrounds/animations/background-color-animation-with-table3.html.ini
--- a/testing/web-platform/meta/css/css-backgrounds/animations/background-color-animation-with-table3.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/css-backgrounds/animations/background-color-animation-with-table3.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,13 +1,12 @@
 [background-color-animation-with-table3.html]
   expected:
-    if not webrender and (os == "win") and debug and (processor == "x86_64"): ["TIMEOUT", "PASS"]
-    if not webrender and (os == "win") and not debug and (processor == "x86"): ["TIMEOUT", "PASS"]
-    if not webrender and (os == "win") and not debug and (processor == "x86_64"): ["TIMEOUT", "PASS"]
-    if webrender and (os == "linux") and debug and swgl: ["TIMEOUT", "PASS"]
-    if webrender and (os == "linux") and debug and not swgl: PASS
-    if webrender and (os == "win") and not swgl and not debug: ["TIMEOUT", "PASS"]
-    if not webrender and (os == "android") and not debug: ["TIMEOUT", "PASS"]
-    if not webrender and (os == "linux") and debug: TIMEOUT
-    if webrender and (os == "win") and swgl: PASS
-    if webrender and (os == "android"): PASS
+    if (os == "win") and not fission and not swgl and not debug and not webrender and (processor == "x86_64"): ["TIMEOUT", "PASS"]
+    if (os == "win") and not fission and not swgl and debug and not webrender and (processor == "x86_64"): ["TIMEOUT", "PASS"]
+    if (os == "linux") and not debug and fission: ["TIMEOUT", "PASS"]
+    if (os == "linux") and debug and not swgl: PASS
+    if (os == "win") and not fission and swgl: PASS
+    if (os == "android") and not webrender and not debug: ["TIMEOUT", "PASS"]
+    if (os == "android") and webrender: PASS
+    if (os == "mac") and not debug: ["TIMEOUT", "PASS"]
+    if (os == "mac") and debug: ["TIMEOUT", "PASS"]
     [PASS, TIMEOUT]